posthog/hogql/database/s3_table.py

```diff
@@ -50,6 +53,36 @@ def return_expr(expr: str) -> str:
 
         return return_expr(expr)
 
+    # Azure
+    if ".blob.core.windows.net/" in url:
+        regex_result = re.search(r"(https:\/\/.+\.blob\.core\.windows\.net)\/(.+?)\/(.*)", url)
```

Comment:
I think we may need to white label these on clickhouse configs too? https://github.com/PostHog/ansible/blob/main/eu/clickhouse/config/config.xml#L78

Reply:
https://github.com/PostHog/posthog-cloud-infra/pull/2891 🥳